src/main/java/online/carsharing/repository/user/UserChatIdRepository.java

```diff
@@ -0,0 +1,14 @@
+package online.carsharing.repository.user;
+
+import java.util.List;
+import online.carsharing.entity.UserChatId;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface UserChatIdRepository extends JpaRepository<UserChatId, Long> {
+    @Query("SELECT uc.chatId FROM UserChatId uc")
+    List<Long> findAllChatIds();
+    
+    @Query("SELECT uc FROM UserChatId uc WHERE uc.userId = :userId")
+    UserChatId findByUserId(Long userId);

```

Comment:
Is it possible to make this query using spring build method? Because it seems possible